api/functions/process_dataset.py

```diff
@@ -80,13 +122,15 @@ def process_dataset_file(event, context):
          context (google.cloud.functions.Context): Metadata of triggering event.
 
     """
-    import base64
-
-    print("""This Function was triggered by messageId {} published at {} to {}
-    """.format(context.event_id, context.timestamp, context.resource["name"]))
-
-    data = base64.b64decode(event['data']).decode('utf-8')
-    print(f'Event data: {data}')
+    print(
+        """This Function was triggered by messageId {} published at {} to {}
```

Comment:
Any reason not to use f-strings here?

Reply:
Absolutely not- I just copied this without styling it. Fixed in next revision.